.github/workflows/development.yml

```diff
@@ -66,23 +65,23 @@ jobs:
         run: git diff --exit-code
 
       - name: Run tests
-        if: matrix.node != '12' || matrix.os != 'ubuntu-latest'
+        if: matrix.node != '14' || matrix.os != 'ubuntu-latest'
         uses: nick-invision/retry@v2
         with:
           timeout_minutes: 20
           max_attempts: 3
           command: npm run test
 
       - name: Run coverage
-        if: matrix.node == '12' && matrix.os == 'ubuntu-latest'
+        if: matrix.node != '14' || matrix.os == 'ubuntu-latest'
```

Comment:
You changed the meaning here by going from AND to OR

Reply:
Might be better to use 16, since that is the current LTS release and won't expire for longer